crates/matrix-sdk/tests/integration/client.rs

```diff
@@ -1249,7 +1247,7 @@ async fn test_rooms_stream() {
     assert!(client.get_room(room_id_3).is_some());
 
     // We receive 3 diffs…
-    assert_let!(Some(diffs) = rooms_stream.next().await);
+    assert_let!(Some(diffs) = futures_util::StreamExt::next(&mut rooms_stream).await);
```

Comment:
What's this about? Seems to be unrelated?

Reply:
I made this change because the compiler was complaining about not being able to decide on which import to use for some reason and I should make it explicit. The `use` above didn't seem to help. I'll try reverting these.

Reply:
Yep, not needed anymore. I don't know what happened when I tried this before.

Reply:
Removed in https://github.com/matrix-org/matrix-rust-sdk/pull/4705/commits/deaaca1aef4448add7ed1cce54db35bb97c6b610